src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -3735,6 +3772,65 @@ class CgroupUtils(object):
                 raise
         return self.assigned_resources is not None
 
+    def add_jobid_to_cgroup_jobs(self, jobid):
+        pbs.logmsg(pbs.EVENT_DEBUG4, 'Adding jobid %s to cgroup_jobs' % jobid)
+        try:
+            with open(self.cgroup_jobs_file, 'a') as f:
+                f.write(jobid + ' ')
+        except IOError:
+            pbs.logmsg(pbs.EVENT_DEBUG, 'Failed to open cgroup_jobs file.')
```

Comment:
The method will still return true when an IOError is encountered. Is this what you want?

Reply:
No that's not what I wanted. Thanks for catching that. Fixed.